RELEASE-NOTES.md

```diff
@@ -1,4 +1,8 @@
 ## RELEASE NOTES
+
+### Version 4.1.6-Title-Display-Error
+**EUI-4377** Title-Display-Error
```

Comment:
Only comment I have is to give more info here e.g. removing unwanted HTML from title or similar

Reply:
Change done, resolving conversation